loki.yml

```diff
@@ -39,7 +39,10 @@ schema_config:
   - from: 2018-04-15
     # Which store to use for the index.
     store: boltdb
-    object_store: filesystem
+    # this is for storing the log chunks locally
+    #object_store: filesystem
+    # uncomment bellow to store the chunks in S3
+    object_store: s3
```

Comment:
I would rather keep this file working for local chunks + indices, and add a separate `loki-s3.yml` using S3 for chunks. Alternatively we can rename `loki.yml` into `loki-local.yml`.

Reply:
I split the config as you suggested.